### PR TITLE
sort brokers for rolling restart

### PR DESCRIPTION
**Pull request by adyach**
sort brokers for rolling restart

### CODE REVIEW

**Comment by adyach** [comment]
👍

**Comment by Kunal-Jha** [comment]
👍

File: bubuku/features/rolling_restart.py [hunk 1]
```diff
@@ -309,7 +309,7 @@ def run(self):
                           'scalyr_region': self.state_context.cluster_config.get_scalyr_region(),
                           'kms_key_id': self.state_context.cluster_config.get_kms_key_id(),
                           'cool_down': self.state_context.cool_down}
-                next_broker_id = list(self.state_context.restart_assignment.keys())[0]
+                next_broker_id = sorted(list(self.state_context.restart_assignment.keys())[0])
```
Comment (ferbncode) [inline bubuku/features/rolling_restart.py]: Should it be `sorted(list(self.state_context.restart_assignment.keys()))[0]`?

**Comment by rcillo** [comment]
👍

**Comment by ferbncode** [comment]
:+1:

**Pull request merged**